src/OpenTelemetry.AutoInstrumentation/Configurations/TracerInstrumentation.cs

```diff
@@ -127,5 +127,10 @@ internal enum TracerInstrumentation
     /// <summary>
     /// Oracle Managed Data Access (Core) instrumentation
     /// </summary>
-    OracleMda = 21
+    OracleMda = 21,
+
+    /// <summary>
+    /// RabbitMQ 6 instrumentation
+    /// </summary>
+    RabbitMq6 = 22
```

Comment:
Do you think that we need to distinguish between RabbitMq6 and RabbtiMq7+ (with native support)?
Any chance to cover it by one configuration?

Reply:
I guess plugable drivers are very rare, so 1 setting should be enough.

Reply:
renamed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3533/commits/dae32c2d7fd9fb016eef7d7773d0f1aa3e64d7a9